src/main/kotlin/app/revanced/patches/all/connectivity/wifi/spoof/patch/SpoofWifiPatch.kt

```diff
@@ -0,0 +1,210 @@
+package app.revanced.patches.all.connectivity.wifi.spoof.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
+import app.revanced.util.patch.AbstractTransformInstructionsPatch
+import app.revanced.util.patch.IMethodCall
+import app.revanced.util.patch.fromMethodReference
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.iface.ClassDef
+import org.jf.dexlib2.iface.Method
+import org.jf.dexlib2.iface.instruction.Instruction
+import org.jf.dexlib2.iface.instruction.formats.Instruction35c
+import org.jf.dexlib2.iface.reference.MethodReference
+import java.util.*
+
+private typealias InstructionInfo = Triple<SpoofWifiPatch.MethodCall, Instruction35c, Int>
```

Comment:
This alias is duplicate of another except for the first generic type. Aliases have generic type parameter specification so you can move this alias to a common place as package internal the following way:
```suggestion
internal typealias InstructionInfo<T: IMethodCall> = Triple<T, Instruction35c, Int>
```

Reply:
Also, a suitable name can be used instead as well, because the triple only considers instructions of type 35c.

Reply:
This would also allow not having to deconstruct the triple to pass it to replaceInvokeVirtualWithIntegrations, as you can then use the typealias as a parameter because it is package private

Reply:
It says `Bounds are not allowed on type alias parameters`. For now I settled with
```kotlin
internal typealias Instruction35cInfo = Triple<IMethodCall, Instruction35c, Int>
```
but I could also use
```kotlin
internal typealias InstructionInfo<T> = Triple<T, Instruction35c, Int>
```